README.md

```diff
@@ -49,6 +49,14 @@ $ docker build services/eventsapi/. -t $REPOHOST/moodle-eventsapi:$TAG
 $ docker push $REPOHOST/moodle-eventsapi:$TAG
 ```
 
+If you do a `kubectl get po` before the push, you can look at the status of the k8s pods. 
+If you find that a pod has errored during setup or its status is `CrashLoopBackOff`, then this could be an issue with the differences in architectures between where the image was created, and where the container is being deployed. 
+
+If the image is being created on machine with ARM-64 architecture and the container deployed on an intel processer, use this to re-build the docker image.  
+```bash
+$ docker build . --platform=linux/amd64 --build-arg MOODLE_VERSION=master -f moodle/Dockerfile -t $REPOHOST/moodle:$TAG
+```
+
```

Comment:
It occurs to me that instead of this content this we can / should just change lines 46 and 48 above to be the following since adding the `--platform` argument would make things work in all environments:

```bash
$ docker build . --platform=linux/amd64 -f moodle/Dockerfile -t $REPOHOST/moodle:$TAG
$ docker push $REPOHOST/moodle:$TAG
$ docker build services/eventsapi/. --platform=linux/amd64 -t $REPOHOST/moodle-eventsapi:$TAG
```

Reply:
😅 great point. That would certainly be easier.